src/main/resources/db/changelog/db.changelog-1.0.0.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet id="create-project-structure" author="MrRevo">
+        <createTable tableName="user">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" primaryKeyName="user_id_pk"/>
+            </column>
+            <column name="key_cloak_id" type="uuid" />
+            <column name="account_type" type="varchar" />
+            <column name="github_profile_url" type="varchar" />
+            <column name="profile_picture_url" type="varchar" />
+        </createTable>
+        <createTable tableName="tag">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" primaryKeyName="tag_id_pk"/>
+            </column>
+            <column name="name" type="varchar" />
+        </createTable>
+        <createTable tableName="user_tags">
+            <column name="user_id" type="bigint"/>
+            <column name="tag_id" type="bigint"/>
+        </createTable>
+        <addForeignKeyConstraint baseTableName="user_tags" baseColumnNames="user_id" constraintName="user_id_fk" referencedTableName="user"
+                                 referencedColumnNames="id"/>
+        <addForeignKeyConstraint baseTableName="user_tags" baseColumnNames="tag_id" constraintName="tag_id_fk" referencedTableName="tag"
+                                 referencedColumnNames="id"/>
```

Comment:
jeszcze onDelete i onUpdate dodaj proszę

Reply:
Jaka ma byc operacja dla delete i update?

Reply:
zależy jak chcesz żeby to się zachowywało, pewnie RESTRICT